devspaces/aggregate/overlays/low-idle/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../default
+
+patches:
```

Comment:
I would suggest keeping these kinds of configurations in the instance overlays and just using the aggregate to reference an instance and a operator overlay.